contributors/guide/pull-requests.md

```diff
@@ -302,34 +313,228 @@ For more information, see [squash commits](./github-workflow.md#squash-commits).
+being done.
 
-Sometimes we need to remind each other of core tenets of software design - Keep It Simple, You Aren't Gonna Need It, Minimum Viable Product, and so on. Adding a feature "because we might need it later" is antithetical to software that ships. Add the things you need NOW and (ideally) leave room for things you might need later - but don't implement them now.
+Commit messages are comprised of two parts; the subject and the body.
 
-## 8. It's OK to Push Back
+The subject is the first line of the commit message and is often the only part
+of the commit message needed for small changes. These are often done as "one
+liners "with the `-m` or `--message` flag.
 
-Sometimes reviewers make mistakes. It's OK to push back on changes your reviewer requested. If you have a good reason for doing something a certain way, you are absolutely allowed to debate the merits of a requested change. Both the reviewer and reviewee should strive to discuss these issues in a polite and respectful manner.
+The commit message body is the portion of text below the subject when you run 
+`git commit` without the `-m` flag which will open the commit message for editing
+in your [preferred editor].
 
-You might be overruled, but you might also prevail. We're pretty reasonable people. Mostly.
+```
+This is the commit message subject
+
+Any text here is the commit message body
+Some text
+Some more text
+...
+
+# Please enter the commit message for your changes. Lines starting
+# with '#' will be ignored, and an empty message aborts the commit.
+#
+# On branch example
+# Changes to be committed:
+#   ...
+#
+```
+
+Use these guidelines below to help craft a well formatted commit message. These
+can be largely attributed to the previous work of [Chris Beams], [Tim Pope],
+[Scott Chacon] and [Ben Straub].
+
+- [Try to keep the subject line to 50 characters or less; do not exceed 72 characters](#try-to-keep-the-subject-line-to-50-characters-or-less-do-not-exceed-72-characters)
+- [The first word in the commit message subject should be capitalized unless it starts with a lowercase symbol or other identifier](#the-first-word-in-the-commit-message-subject-should-be-capitalized-unless-it-starts-with-a-lowercase-symbol-or-other-identifier)
+- [Do not end the commit message subject with a period](#do-not-end-the-commit-message-subject-with-a-period)
+- [Use imperative mood in your commit message subject](#use-imperative-mood-in-your-commit-message-subject)
+- [Add a single blank line before the commit message body](#add-a-single-blank-line-before-the-commit-message-body)
+- [Wrap the commit message body at 72 characters](#wrap-the-commit-message-body-at-72-characters)
+- [Do not use GitHub keywords or (@)mentions within your commit message](#do-not-use-github-keywords-or-mentions-within-your-commit-message)
+- [Use the commit message body to explain the _what_ and _why_ of the commit](#use-the-commit-message-body-to-explain-the-what-and-why-of-the-commit)
+
```

Comment:
Could this list be split or ordered by importance, eg: first content needs (what, why, concise summary), second formatting needs (blank space after subject line), and formatting desires (first letter Capitalized, no trailing period).  The "what and why" is at the end of the list, yet is critical, while some of the formatting desires, while justifiable, are notably less critical, imho.

Reply:
My thought on the ordering was a sort of what you encounter first approach hitting the relevant sections for the subject, how to add the body, what things if found will prevent it from merging. Maybe re-order with the what/why at the top?

Reply:
I left it as is, but expanded a little further on it in the subject line topic and referenced that section.

Reply:
The latest commit really helps!  Thanks!